### PR TITLE
docs: update figma kit page with card icons

### PR DESCRIPTION
**Pull request by laurenmrice**
**Changed**
- Update the Figma kit page to include Figma icons on each resource card.

---

**Testing**
- Go to `Designing` --> `Design kits` --> `Figma` tab
  - Make sure resource card icons appear correctly.

### CODE REVIEW

**Comment by laurenmrice** [comment]
My preview keeps failing because I think something is wrong with the way I am trying to embed the figma icon in the resource cards. Anyone know how to fix it? 👀

**Comment by laurenmrice** [comment]
@aagonzales It wasn't working with the analytics code, so I tried taking it out, but maybe it is just the svg vs. png difference. I will give that a go!

File: src/pages/designing/kits/figma.mdx [hunk 1]
```diff
@@ -55,40 +58,58 @@ Then in the Libraries modal that appears, find the team called
...
+
     </ResourceCard>
+
   </Column>
   <Column colMd={4} colLg={4} noGutterSm>
     <ResourceCard
-      onClick={() => fathom.trackGoal('T7D1HJ3L', 0)}
       subTitle="Gray 10 theme"
       href="https://www.figma.com/file/i6y80WeotmhmbWxpmcffz7/Gray-10-Theme---Carbon-Design-System">
+      actionIcon="launch">
```
Comment (jnm2377) [inline src/pages/designing/kits/figma.mdx:82]: Looks like the problem is that you forgot to delete the old closing tag when adding the new line. You can add back the onClick.
```suggestion
      href="https://www.figma.com/file/i6y80WeotmhmbWxpmcffz7/Gray-10-Theme---Carbon-Design-System"
      actionIcon="launch">
```

File: src/pages/designing/kits/figma.mdx [hunk 2]
```diff
@@ -55,40 +58,58 @@ Then in the Libraries modal that appears, find the team called
...
+
     </ResourceCard>
+
   </Column>
   <Column colMd={4} colLg={4} noGutterSm>
     <ResourceCard
-      onClick={() => fathom.trackGoal('LYFJTPDE', 0)}
       subTitle="Gray 90 theme"
       href="https://www.figma.com/file/fCucrHE8tdGDVJX7ODqajd/Gray-90-Theme---Carbon-Design-System">
+      actionIcon="launch">
```
Comment (jnm2377) [inline src/pages/designing/kits/figma.mdx:93]:
```suggestion
      href="https://www.figma.com/file/fCucrHE8tdGDVJX7ODqajd/Gray-90-Theme---Carbon-Design-System"
      actionIcon="launch">
```
same as above

File: src/pages/designing/kits/figma.mdx [hunk 3]
```diff
@@ -55,40 +58,58 @@ Then in the Libraries modal that appears, find the team called
...
+
     </ResourceCard>
+
   </Column>
   <Column colMd={4} colLg={4} noGutterSm>
     <ResourceCard
-      onClick={() => fathom.trackGoal('3XH0SIBJ', 0)}
       subTitle="Gray 100 theme"
       href="https://www.figma.com/file/TckQzGe3bNxHPLoRhbXFal/Gray-100-Theme---Carbon-Design-System">
+      actionIcon="launch">
```
Comment (jnm2377) [inline src/pages/designing/kits/figma.mdx:104]:
```suggestion
      href="https://www.figma.com/file/TckQzGe3bNxHPLoRhbXFal/Gray-100-Theme---Carbon-Design-System"
      actionIcon="launch">
```

File: src/pages/designing/kits/figma.mdx (hunk 1 above)
Comment (laurenmrice) [reply src/pages/designing/kits/figma.mdx:82]: Oh thank you @jnm2377 ! Hopefully this fixes it!

File: src/pages/designing/kits/figma.mdx [hunk 4]
```diff
@@ -55,40 +58,58 @@ Then in the Libraries modal that appears, find the team called
...
+
+To preview the four Carbon themes, visit these view-only links.
 
 <Row className="resource-card-group">
   <Column colMd={4} colLg={4} noGutterSm>
     <ResourceCard
-      onClick={() => fathom.trackGoal('P0OEN9TS', 0)}
       subTitle="White theme"
       href="https://www.figma.com/file/Vzz8k68Pqk5HfaTdQOQrGu/White-Theme---Carbon-Design-System">
+      actionIcon="launch">
```
Comment (jnm2377) [inline src/pages/designing/kits/figma.mdx:71]:
```suggestion
      href="https://www.figma.com/file/Vzz8k68Pqk5HfaTdQOQrGu/White-Theme---Carbon-Design-System"
      actionIcon="launch">
```

File: src/pages/designing/kits/figma.mdx [hunk 5]
```diff
@@ -55,40 +58,58 @@ Then in the Libraries modal that appears, find the team called
...
+
     </ResourceCard>
+
   </Column>
   <Column colMd={4} colLg={4} noGutterSm>
     <ResourceCard
-      onClick={() => fathom.trackGoal('T7D1HJ3L', 0)}
       subTitle="Gray 10 theme"
       href="https://www.figma.com/file/i6y80WeotmhmbWxpmcffz7/Gray-10-Theme---Carbon-Design-System">
+      actionIcon="launch">
```
Comment (jnm2377) [inline src/pages/designing/kits/figma.mdx:82]:
```suggestion
      href="https://www.figma.com/file/i6y80WeotmhmbWxpmcffz7/Gray-10-Theme---Carbon-Design-System"
      actionIcon="launch">
```

File: src/pages/designing/kits/figma.mdx [hunk 6]
```diff
@@ -55,40 +58,58 @@ Then in the Libraries modal that appears, find the team called
...
+
     </ResourceCard>
+
   </Column>
   <Column colMd={4} colLg={4} noGutterSm>
     <ResourceCard
-      onClick={() => fathom.trackGoal('LYFJTPDE', 0)}
       subTitle="Gray 90 theme"
       href="https://www.figma.com/file/fCucrHE8tdGDVJX7ODqajd/Gray-90-Theme---Carbon-Design-System">
+      actionIcon="launch">
```
Comment (jnm2377) [inline src/pages/designing/kits/figma.mdx:93]:
```suggestion
      href="https://www.figma.com/file/fCucrHE8tdGDVJX7ODqajd/Gray-90-Theme---Carbon-Design-System"
      actionIcon="launch">
```

**Review by jnm2377 — COMMENTED** [review]
You can't have 2 closing carets. That's why it's breaking. You're closing the `<ResourceCard>` after the `href` lines, and then adding another closing caret after `actionIcon="launch"`.

**Comment by jnm2377** [comment]
Gatsby theme can be really finicky with indentation. I pushed a fix, so it should be working now.

**Review by jnm2377 — APPROVED** [review]

**Review by aagonzales — APPROVED** [review]

**Comment by aagonzales** [comment]
@tay1orjones looks like you need to review this to unblock the merge.

**Review by tay1orjones — APPROVED** [review]
Sorry! Meant to put an approve on this earlier

**Pull request merged**